react/src/components/atoms/forms/InputCheckBox/style.scss

```diff
@@ -0,0 +1,17 @@
+@import 'global';
+$assets-path: '../../../../assets';
+@import '01-atoms/input-checkbox';
```

Comment:
is there a reason you put this here vs. in the higher level shared assets folder? can we move it up?

Reply:
because this path is relative to the css file, this path is react specific.